SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestTearOff/SpatialTestTearOff.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialFunctionalTest.h"
+#include "SpatialTestTearOff.generated.h"
+
+class AReplicatedTearOffActor;
+class AReplicatedTestActorBase;
+
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API ASpatialTestTearOff : public ASpatialFunctionalTest
+{
+	GENERATED_BODY()
+
+public:
+	ASpatialTestTearOff();
+
+	virtual void BeginPlay() override;
+
+	// Variable used to reference the ReplicatedTearOffActor placed in the map.
+	AReplicatedTearOffActor* StartupTearOffActor;
+
+	// Variable used to reference the spawned AReplicatedTestActorBase.
+	AReplicatedTestActorBase* SpawnedReplicatedActorBase;
```

Comment:
Since this is will be the only reference to the newly spawned actor from SpawnActor(), I think it should be marked UPROPERTY() to prevent garbage collection?

Reply:
To be honest, I do not know much about UE's garbage collection, if that is the case, then I will definitely add that. What do you think about this @ImprobableNic @mironec ?

Reply:
I do not think this is necessary. Spawned actors should be referenced in the Level.